tools/cleanup/cleanup.sh

```diff
@@ -0,0 +1,285 @@
+#!/bin/bash 
+
+# Cleans up all Trilio resources from the k8s cluster
+# IMP - Runs on the sourced kubeconfig (refers to .kube/config)
+# Please make sure that correct kubeconfig is used or sourced
+# Deleted resources can not be recovered
+
+CLEANUP_RUN_SUCCESS=true
+
+check_if_ocp() {
+  # Check if the k8s cluster is upstream or OCP
+  local is_ocp="N"
+  if (kubectl api-resources | grep -q openshift.io); then
+    is_ocp="Y"
+  fi
+  echo "${is_ocp}"
+}
+
+delete_tvk_res(){
+  # Same for OCP & Upstream
+  # Check all the namespaces for restores, delete the restores 
+  local exit_status=0
+  for res in ${TVK_resources}
+  do
+    if (kubectl get "${res}" -A --no-headers 2>/dev/null)
+    then
+      # Fetch non-deuplicate namespace for the given resource
+      for ns in $(kubectl get "${res}" -A --no-headers 2>/dev/null | awk '{print $1}'| uniq) 
+      do
+        # Fetch given resource name
+        for name in $(kubectl get "${res}" -n "${ns}" --no-headers 2>/dev/null | awk '{print $1}'| uniq)
+        do 
+          # Delete 
+          echo "kubectl delete ${res} ${name} -n ${ns} "
+          kubectl delete "${res}" "${name}" -n "${ns}"
```

Comment:
simple `delete` may not work in all cases, will need to remove finalizers then patch resource. Some deletion calls might get blocked form TVK webhook so need to remove TVK's webhookConfigs first.

Reply:
I would address this in next release as this will change the logic. Trying to get the initial cleanup plugin out and then we will resolve these issues.